engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/runtime/RuntimeCurrentExecutionEnvironmentNode.java

```diff
@@ -0,0 +1,17 @@
+package org.enso.interpreter.node.expression.builtin.runtime;
+
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.state.State;
+
+@BuiltinMethod(
+    type = "Runtime",
+    name = "current_execution_environment",
```

Comment:
This method shall not exists from my API designer perspective as it bypasses the `Context`. I assume @jdunkerley wants the method to do production/non-production property configuration, but we should offer other means for that - not this method.

Reply:
We agreed that `Execution_Environment` is really an alias for `Text` and I believe I'm follow the design doc so I don't really understand the comment.

Reply:
There is a story about [magical strings](http://wiki.apidesign.org/wiki/MagicalStrings). I have a feeling the returned textual value has the potential to provide "magical" insights into something that should rather be hidden. 

> ...I don't expect this to be used.

Personally I'd rather not expose such API, especially if there is no use-case for it.